src/main/java/domain/Command.java

```diff
@@ -0,0 +1,37 @@
+package domain;
+
+public class Command {
+
+    private static final int COMMAND_MIN_SIZE_INCLUSIVE = 1;
+    private static final int COMMAND_MAX_SIZE_INCLUSIVE = 5;
+    private static final String COMMAND_SIZE_ERROR_MESSAGE = "명령은 1 ~ 5 글자여야 합니다.";
+    private static final String ALL_INCLUSIVE = "all";
+
+    private final String command;
+
+    public Command(String command) {
+        validate(command);
+        this.command = command;
+    }
+
+    private void validate(String command) {
+        if (isOutOfRange(command)) {
+            throw new IllegalArgumentException(COMMAND_SIZE_ERROR_MESSAGE);
+        }
+    }
+
+    private boolean isOutOfRange(String command) {
+        return !(COMMAND_MIN_SIZE_INCLUSIVE <= command.length()
+                && command.length() <= COMMAND_MAX_SIZE_INCLUSIVE);
+    }
+
+    public boolean isAllCommand() {
+        return command.equals(ALL_INCLUSIVE);
```

Comment:
```suggestion
        return ALL_INCLUSIVE.equals(command);
```
NullPointerException이 발생할 수 있어서 항상 상수 쪽에서 equals메서드를 호출해주는것이 좋습니다

Reply:
꿀팁 감사합니다!